channeldb/channel.go

```diff
@@ -360,6 +364,11 @@ func (c ChannelType) HasScidAliasFeature() bool {
 	return c&ScidAliasFeatureBit == ScidAliasFeatureBit
 }
 
+// IsTaproot returns true if the channel is using taproot features.
+func (c ChannelType) IsTaproot() bool {
+	return c&SimpleTaprootFeatureBit == SimpleTaprootFeatureBit
+}
+
```

Comment:
can we add this diff to this PR just to block taproot channel commitments from being sent to the tower client?
(just as a precaution incase https://github.com/lightningnetwork/lnd/pull/7733 does not get in in-time for the release that includes tap chans)
```
diff --git a/peer/brontide.go b/peer/brontide.go
index 9ea85edd1..399c66c34 100644
--- a/peer/brontide.go
+++ b/peer/brontide.go
@@ -956,7 +956,20 @@ func (p *Brontide) addLink(chanPoint *wire.OutPoint,
 	// okay if the clients are disabled altogether and these values are nil,
 	// as the link will check for nilness before using either.
 	var towerClient htlcswitch.TowerClient
-	if chanType.HasAnchors() {
+	if chanType.IsTaproot() {
+		// Leave the tower client as nil for now until the tower client
+		// has support for taproot channels.
+
+		// If the user has activated the tower client, then add a log
+		// to explain that any taproot channel updates wil not be
+		// backed up to a tower.
+		if p.cfg.TowerClient != nil {
+			p.log.Debugf("Updates for channel %s will not be "+
+				"backed up to a watchtower as watchtowers "+
+				"are not yet taproot channel compatible",
+				chanPoint)
+		}
+	} else if chanType.HasAnchors() {
 		towerClient = p.cfg.AnchorTowerClient
 	} else {
 		towerClient = p.cfg.TowerClient

```

Reply:
Sure, tacked on the commit here as it's convenient (could be later in the series tho).